src/download.rs

```diff
@@ -110,41 +111,28 @@ pub fn parse_linux_version(version: &str) -> Option<String> {
     .map(|s| s.to_string())
 }
 
-/// Invoke cURL on the command line to download a file, returning it as a string.

```

Comment:
Let's keep both this function and `curl_to_file` for now, but mark them both as [`#[deprecated]`](https://doc.rust-lang.org/reference/attributes/diagnostics.html#the-deprecated-attribute). And then in the next major version release we can fully remove them.